canvas_modules/common-canvas/src/palette/palette-flyout-content-search.jsx

```diff
@@ -52,7 +52,7 @@ class PaletteFlyoutContentSearch extends React.Component {
 			// palette-flyout-search id added for hopscotch tours
 			<div className="palette-flyout-search-container" id="palette-flyout-search">
 				<Search
-					id="palette-flyout-search"
+					id="palette-flyout-search-input"
```

Comment:
Keep `id="palette-flyout-search"` on line 55 and try adding `id="palette-flyout-search-container"` on line 53